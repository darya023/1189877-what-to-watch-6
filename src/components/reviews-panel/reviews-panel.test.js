import React from 'react';
import {render} from '@testing-library/react';
import ReviewsPanel from './reviews-panel';

it(`Should ReviewsPanel render correctly`, () => {
  const {container} = render(
      <ReviewsPanel
        reviews={[]}
      />
  );
  expect(container).toMatchSnapshot();
});
