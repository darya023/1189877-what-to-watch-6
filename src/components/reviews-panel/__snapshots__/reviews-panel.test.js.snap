// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ReviewsPanel render correctly 1`] = `
<div>
  <div
    class="movie-card__reviews movie-card__row"
  >
    <div
      class="movie-card__reviews-col"
    >
      <div
        class="review"
      >
        There are no reviews yet
      </div>
    </div>
  </div>
</div>
`;
